test/policy/templates/project-authn-def.template.sh.yml

```diff
@@ -20,6 +20,13 @@ cat << EOL
         kubernetes/authentication-container-name: cyberark-secrets-provider
         openshift: "true"
 
+    # this host will not have permissions on Conjur secrets to test this use-case
+    - !host
+      id: ${TEST_APP_NAMESPACE_NAME}/service_account/${TEST_APP_NAMESPACE_NAME}-sa
```

Comment:
why do we have this long path here? and why service_account instead of just a regular host?